django_dynamic_theme/models.py

```diff
@@ -190,9 +193,15 @@ def export(self) -> str:
         """
         Exports all listed configurations as string in SCSS format.
         """
-        return f"""body {{{self.background.export()}}}
-{self.media_gallery.export()}
-{self.navbar.export()}"""
+        background = self.background.export()
+        try: mediagallery = self.media_gallery.export()
```

Comment:
Can you check for None instead of try?

Reply:
yes will do this evening, apologies for that